### DISCUSSION TITLE
How to create new congregation?

### DISCUSSION THREAD

**Discussion by taras-lysyi** [sws2apps/sws2apps-api#346] [Q&A] [answered]
Hello! I want to try your app but cannot create a congregation. How to do it?

https://user-images.githubusercontent.com/48454662/211206303-55cfaad1-4e15-4be5-909f-9b1b442d819d.mov

**Comment by rhahao** [comment, marked as answer]
Hello @taraslis453, first of all, I’m very sorry that it’s now that I see this message, due to incorrect settings on my notification.
Basically, you have to type and search for a valid congregation name.
